src/vue/modules/requests/asset-creation-requests/store/index.js

```diff
@@ -0,0 +1,62 @@
+import { AssetCreationRequest } from '../wrappers/asset-creation-request'
+
+import { base } from '@tokend/js-sdk'
+
+import { types } from './types'
+import { api } from '../_api'
+
+const HORIZON_VERSION_PREFIX = 'v3'
+
+export const state = {
+  accountId: '',
+  assetCreationRequests: [],
+}
+
+export const mutations = {
+  [types.SET_ACCOUNT_ID] (state, accountId) {
+    state.accountId = accountId
+  },
+  [types.SET_ASSET_CREATION_REQUESTS] (state, assetCreationRequests) {
+    state.assetCreationRequests = assetCreationRequests
+  },
+  [types.CONCAT_ASSET_CREATION_REQUESTS] (state, assetCreationRequests) {
+    state.assetCreationRequests = state.assetCreationRequests
+      .concat(assetCreationRequests)
+  },
+}
+
+export const actions = {
+  [types.LOAD_ASSET_CREATION_REQUESTS] ({ getters }) {
+    return api().getWithSignature(`/${HORIZON_VERSION_PREFIX}/create_asset_requests`, {
```

Comment:
Guys made an API more consistent so we can make the client too.

if it's called `create_asset_request` on API, then we should call it the same way on clients. Same for every other request type

Reply:
Renamed:
- `asset-creation-requests` -> `create-asset-requests`
- `asset-update-requests` -> `update-asset-requests`
- `sale-creation-requests` -> `create-sale-requests`